src/sage/rings/localization.py

```diff
@@ -954,11 +967,12 @@ def fraction_field(self):
 
         EXAMPLES::
 
-            sage: R.<a> = GF(5)[]                                                       # optional - sage.rings.finite_rings
-            sage: L = Localization(R, (a**2 - 3, a))                                    # optional - sage.rings.finite_rings
-            sage: L.fraction_field()                                                    # optional - sage.rings.finite_rings
+            sage: # needs sage.rings.finite_rings
+            sage: R.<a> = GF(5)[]
```

Comment:
Is this correct?

Reply:
Fixed in 59e556e - a tag is needed, but sage.libs.pari is a better choice.